x-pack/plugins/observability_solution/infra/public/alerting/inventory/components/metric.tsx

```diff
@@ -56,6 +56,14 @@ interface Props {
     | 'rightDown';
 }
 
+type V2MetricType = 'txV2' | 'rxV2' | 'cpuTotal';
```

Comment:
I wonder if we should rename `cpuTotal` to `cpuV2`. Does it make sense for consistency's sake?

Reply:
Thanks for the suggestion, it makes sense 👍  Renamed ✅